ann_benchmarks/plotting/metrics.py

```diff
@@ -1,6 +1,13 @@
 from __future__ import absolute_import
 import numpy as np
 
+def compute_knn(dataset_distances, run_distances, count, epsilon=1e-10):
+    threshold = dataset_distances[:,count - 1] + epsilon
+    recalls = np.zeros(len(run_distances))
+    for i in range(len(run_distances)):
+        recalls[i] = np.count_nonzero(run_distances[i] <= threshold[i])
+    return recalls
+
```

Comment:
Just out of curiosity: How much faster is that? This should probably go upstream immediately :-)

Reply:
I don't have a precise benchmark, but overall the export was 4 time faster, so I guess it's in that ballpark :)

I didn't change the `knn` function because I didn't want to change the caching code